src/setup/keyboard.c

```diff
@@ -230,7 +230,7 @@ static void ConfigExtraKeys(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
         AddKeyControl(table, "Toggle vert. mouse", &key_togglenovert);
         AddKeyControl(table, "Quick Reverse", &key_reverse);
         }
-        else if (gamemission == heretic)
+        else if (gamemission == heretic || gamemission == hexen)
```

Comment:
It seems that Hexen is still missing the `key_alt_*` bindings which are also set in this section. Would you mind adding them as well in this PR?

Reply:
You're right, the binds show in the setup tool, but they don't work in game.
I'll take a look at it.